src/language-css/utils/stringify-func-param.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const getHighestAncestor = require("./get-highest-ancestor.js");
+
+/**
+ * @param {*} node
+ * @returns {string}
+ */
+function stringifyFuncParam(node) {
```

Comment:
We can also use this here.
https://github.com/prettier/prettier/blob/4d4d2a6fc0844f513ea0c6d5316ec7e37b89df66/src/language-css/parser-postcss.js#L54
I'll change it if you want.